File: prjonlineshopping/src/app/models/Cart.model.ts
export class CartModel {
    CartID: number;
    ProductDescription: string;
    ProductCode: string;
    ProductName: string;
    Quantity: number;
    ProductID: number;
    ProductPrice: number;
    TotalPrice: number;
}

export class UpdateCartModel {
    constructor() {
        this.UserID = 0;
        this.Quantity = 0;
        this.ProductID = 0;
        this.TotalPrice = 0;
    }
    UserID: number;
    Quantity: number;
    ProductID: number;
    TotalPrice: number;
}

File: OnlineShopping/OnlineShopping/Models/CartModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OnlineShopping.Models
{
    public class CartModel
    {
        public int CartID { get; set; }
        public int ProductID { get; set; }
        public string ProductCode { get; set; }
        public string ProductName { get; set; }
        public string ProductDescription { get; set; }
        public float ProductPrice { get; set; }
        public double TotalPrice { get; set; }
        public int Quantity { get; set; }
    }

    public class UpdateCartModel
    {
        public int ProductID { get; set; }
        public int UserID { get; set; }
        public double TotalPrice { get; set; }
        public int Quantity { get; set; }
    }
}